server/routerlicious/packages/services-telemetry/src/lumberFormatters.ts

```diff
@@ -0,0 +1,36 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Lumberjack } from "./lumberjack";
+import { ILumberFormatter } from "./resources";
+import { Lumber } from "./lumber";
+
+export class SanitationFormatter implements ILumberFormatter {
+	private readonly sensitiveKeys = [
+		/cookie/i,
+		/passw(or)?d/i,
+		/^pw$/,
+		/^pass$/i,
+		/secret/i,
+		/token/i,
+		/api[._-]?key/i,
+		/session[._-]?id/i,
+	];
+
+	private readonly redactedStr = "[REDACTED]"
+
+	public transform(lumber: Lumber<string>): void {
+		if (lumber.exception) {
```

Comment:
Check for logLevel.Error ?

Reply:
Warning levels can also include an exception. So either we need to check for both error and warning, or we omit this check as it was done originally